library/objective-c/EnvoyBridgeUtility.h

```diff
@@ -3,7 +3,7 @@
 #import "library/common/types/c_types.h"
 
 static inline envoy_data toNativeData(NSData *data) {
-  if (data == nil) {
+  if (data == nil || data == (id)[NSNull null]) {
```

Comment:
Might make more sense from a readability standpoint to do `(id)data == [NSNull null]` but not a huge deal